snmp/datadog_checks/snmp/data/default_profiles/dell-os10.yaml

```diff
@@ -0,0 +1,231 @@
+extends:
+  - _base.yaml
+  - _generic-host-resources-base.yaml
+  - _generic-ucd.yaml
+#  - dell.yaml  # TODO: check hierarchy
+#  - if32-mib.yaml  # TODO: later (NDM-1933)
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "dell"
+
+sysobjectid:
+  - 1.3.6.1.4.1.674.11000.5000.100.2.*      # OS10 family
+  - 1.3.6.1.4.1.674.11000.5000.100.2.1.1    # s6000on
+  - 1.3.6.1.4.1.674.11000.5000.100.2.1.9    # s4128Ton
+  - 1.3.6.1.4.1.674.11000.5000.100.2.1.20   # s5232Fon
+  - 1.3.6.1.4.1.674.11000.5000.100.2.1.21   # s5248Fon
+
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    table:
+      OID: 1.3.6.1.2.1.25.3.3
+      name: hrProcessorTable
+    symbols:
+      - OID: 1.3.6.1.2.1.25.3.3.1.2
+        name: cpu.usage
+    metric_tags:
+      - tag: hr_processor_frw_id
+        column:
+          OID: 1.3.6.1.2.1.25.3.3.1.1
+          name: hrProcessorFrwID
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.2.1.25.2.3.1.5.1
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.used
+      OID: 1.3.6.1.2.1.25.2.3.1.6.1
+  - MIB: DELLEMC-OS10-CHASSIS-MIB
+    symbol:
+      name: os10ChassisTemp
+      OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.1.3.1.11
+  - MIB: DELLEMC-OS10-CHASSIS-MIB
+    table:
+      OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.1.4
+      name: os10CardTable
+    symbols:
+      - OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.1.4.1.5
+        name: os10CardTemp
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.1.4.1.3
+          name: os10CardDescription
+        tag: os10_card_description
+      - column:
+          OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.1.4.1.9
+          name: os10CardServiceTag
+        tag: os10_card_service_tag
+      - column:
+          OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.1.4.1.4
+          name: os10CardStatus
+        tag: os10_card_status
+        mapping:
+          1: up
+          2: down
+          3: testing
+          4: unknown
+          5: dormant
+          6: not_present
+          7: lower_layer_down
+          8: failed
+  - MIB: DELLEMC-OS10-CHASSIS-MIB
+    table:
+      OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.2.1
+      name: os10PowerSupplyTable
+    symbols:
+      - name: os10PowerSupply
+        constant_value_one: true
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.2.1.1.4
+          name: os10PowerSupplyOperStatus
+        tag: os10_power_supply_oper_status
+        mapping:
+          1: up
+          2: down
+          3: testing
+          4: unknown
+          5: dormant
+          6: not_present
+          7: lower_layer_down
+          8: failed
```

Comment:
I think only tagging with `os10_power_supply_oper_status` since we need a tag to make each row unique.

Usually you can check other columns in `os10PowerSupplyTable` that might help to create metrics with unique set of tags for each table row.

Reply:
Thanks, got it